cpp/src/ice2slice/Gen.cpp

```diff
@@ -610,11 +611,11 @@ Gen::TypesVisitor::visitSequence(const SequencePtr& p)
             }
             out << "<"
                 // TODO the generic argument must be the IceRPC C# mapped type
-                << typeToString(p->type(), p->scope(), false) << ">\")]";
+                << typeToString(p->type(), scope, false) << ">\")]";
```

Comment:
We already store `p->scope()` in a variable.
No need to keep calling it over and over.